pkg/cli/zip_table_registry.go

```diff
@@ -1037,12 +1037,18 @@ var zipSystemTables = DebugZipTableRegistry{
 		},
 	},
 	"system.namespace": {
-		nonSensitiveCols: NonSensitiveColumns{
-			`"parentID"`,
-			`"parentSchemaID"`,
-			"name",
-			"id",
-		},
+		customQueryUnredacted: `SELECT "parentID",
+"parentSchemaID",
+"name",
+"id"
+FROM system.namespace AS OF SYSTEM TIME '-0.1s'
+`,
+		customQueryRedacted: `SELECT "parentID",
+"parentSchemaID",
+"name",
+"id"
+FROM system.namespace AS OF SYSTEM TIME '-0.1s'
+`,
```

Comment:
I think the idea is that all tables which get dumped should be queried AOST and not just the catalog tables, but I may be wrong.

Reply:
We can opt instead for ```BEGIN AS OF SYSTEM TIME ...``` and have this blanket applied within the debug zip command

Reply:
Works for me